Cargo.toml

```diff
@@ -14,3 +14,6 @@ members = [
     "signature_derive",
     "universal-hash",
 ]
+
+[patch.crates-io]
+signature = { path = "./signature" }
```

Comment:
this I'm not sure.

I had this error otherwise:
```
error: failed to select a version for `signature`.
    ... required by package `crypto v0.5.1 (traits/crypto)`
versions that meet the requirements `^2` (locked to 2.2.0) are: 2.2.0

all possible versions conflict with previously selected packages.

  previously selected package `signature v2.3.0-pre.2`
    ... which satisfies dependency `signature = "=2.3.0-pre.2"` of package `async-signature v0.6.0-pre.0 (traits/async-signature)`

failed to select a version for `signature` which could resolve this conflict
```

Reply:
That's fine for now